bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -177,16 +177,20 @@ public String getScreenReaderText() {
     @Nullable
     public String getHtmlScreenReaderText() {
         // this can be used in sightly to compute initial html
-        String screenReaderText = getName();
+        String screenReaderText = "";
         if (AssistPriority.LABEL.equals(assistPriority)) {
```

Comment:
is there a default assist priority value desired now? It does not seems so from this PR.
we should have a default value for screen reader text.

Reply:
Default is empty. (No corresponding property is saved in JCR)